iterative/resource_task.go

```diff
@@ -230,6 +231,19 @@ func resourceTaskBuild(ctx context.Context, d *schema.ResourceData, m interface{
 		}
 	}
 
+	for _, e := range os.Environ() {
+		pair := strings.SplitN(e, "=", 2)
+		if pair[1] != "" && (strings.HasPrefix(pair[0], "CI") ||
+			strings.HasPrefix(pair[0], "GITHUB") ||
+			strings.HasPrefix(pair[0], "BITBUCKET") ||
+			strings.HasPrefix(pair[0], "REPO_TOKEN") ||
+			strings.HasPrefix(pair[0], "CML")) {
+			v[pair[0]] = &pair[1]
+		}
+	}
```

Comment:
```suggestion
	v["CI"] = nil
    v["CI_*"] = nil
	v["CML_*"] = nil
	v["GITHUB_*"] = nil
	v["GITLAB_*"] = nil
	v["BITBUCKET_*"] = nil
	v["REPO_TOKEN"] = nil
```

💡 Tip: you can use the internal glob engine to avoid dealing with doce duplication; `task` will take care of expanding any variables matching the glob.

Reply:
I can fix it